src/ActionModal/components/ActionModal.vue

```diff
@@ -649,7 +649,7 @@ export default {
       this.close()
 
       this.$store.dispatch(`signOut`, this.network)
-      if (this.$route.name !== `portfolio`) this.$router.push(`portfolio`)
+      if (this.$route.name !== `portfolio`) this.$router.push(`/`)
```

Comment:
```suggestion
      if (this.$route.name !== `portfolio`) this.$router.push({ name: 'portfolio' })
```

https://router.vuejs.org/guide/essentials/navigation.html

Reply:
Then we need to change a bunch of routes here

Reply:
OK, actually not 😅 There were two like this for extension. All good